src/app/components/MediaLoader/types.ts

```diff
@@ -208,14 +212,41 @@ export type ClipMediaBlock = {
   };
 };
 
+export type LegacyMediaBlock = {
+  type: 'legacyMedia';
+  content: {
+    id: string;
+    subType: string;
+    format: MediaType;
+    image: {
+      id: string;
+      subType: string;
+      href: string;
+      path: string;
+      height: number;
+      width: number;
+      altText: string;
+      copyrightHolder: string;
+    };
+    aspectRatio: string;
+    live: boolean;
+    href: string;
+    playlist: {
+      format: string;
```

Comment:
Does this also need to include bitrate? https://github.com/bbc/morph-modules/blob/master/news-av-player/src/legacy-media/transformPlaylist/index.js#L6

Reply:
bitrate doesn't seem to be used for anything in the SMP config: http://cookbook.tools.bbc.co.uk/iplayer/customise

Reply:
OK cool. This is also about simplifying the SMP settings, and modernising rather than like-for-like